### ISSUE TITLE
Exclude models from updater backup

### ISSUE THREAD

**Issue by jplitza** [nextcloud/recognize#944]
### Describe the feature you'd like to request

Updating my Nextcloud via webinterface is no longer possible, since the updater tries for ages to backup the models under `apps/recognize/models` (as well as `/apps/recognize/node_modules/@tensorflow/tfjs-node-gpu/deps/lib tensorflow.so` – three times thanks to not supporting symlinks).

### Describe the solution you'd like

Download the models to `data/` instead? That's ignored by the updater AFAIK.

### Describe alternatives you've considered

Modifying/configuring the updater to exclude the path `apps/recognize/models`? Don't know if that's a supported feature.

**Comment by jplitza** [comment]
FTR: I created nextcloud/server#39843 about the symlink support for libtensorflow.so

**Comment by marcelklehr** [comment]
Yes, the path for storing the models is not ideal at the moment. It's the same for the translate, stt_whisper and llm apps. 

a) The models get removed everytime an update comes in and b) they get backupped unnecessarily.

The problem with storing them in data/ is that there is no hook for the uninstall event in nextcloud, which means recognize will not be able to remove the models when it's uninstalled :/

**Comment by NotAFile** [comment]
I don't mind the delay that much personally because I use the CLI updater, but it is a bit unfortunate because I don't have that much storage space available. That means that after one or two updates, I need to go into the backups and manually delete them, or I run completely out of disk.

**Comment by Multigestern** [comment]
Any Updates on this?
My Backups are taking Hours currently.

**Comment by solonovamax** [comment]
Is appdata deleted upon app deletion? I can't tell whether or not that's the case.

Even if it's not deleted, I think it would be *much* better to use appdata and just document how to fully delete it instead of using the app's directory.

for me, the backup process takes *so* long that it actually ends up timing out. 

in fact, it's not *just* the model symlinks which is an issue.
the three largest files for me are:
```
1.1G	recognize.tar.gz
1.2G	models
1.5G	node_modules
```

these end up *massively* bloating the backup size.
